neuromation/cli/formatters/config.py

```diff
@@ -1,35 +1,66 @@
 from sys import platform
+from typing import Dict
 
 from click import style
+from tabulate import tabulate
```

Comment:
It helps format tables with mininal width

Reply:
The only thing I dislike is that `tabulate` installs an executable command as a side effect.
Thus, `pip install neuromation` actually installs `neuro`, `docker-credential-neuro` and `tabulate` console scripts.
Not sure how is it harm, though.

Reply:
OK, I will try to avoid binary install

Reply:
No way (